### PR TITLE
Enhancement/apply or translate

### PR DESCRIPTION
**Pull request by DonWillems**
- Fixed some translations that came up with the introduction of or-translate to or-mwc-input type BUTTON
- Removed i18next from uses of the or-mwc-input type button
- Added or-translate to snackbar and removed i18next from snackbar instances
- Added or-translate to or-mwc-list so that e.g. the top right menu in the manager immediately translates when switching languages
- Applied or-translate to the settings drawers of widgets on the insights page

### CODE REVIEW

**Review by MartinaeyNL — CHANGES_REQUESTED** [review]
In general, I'm struggling with the auto-translating functionality we've added here.
On the 'label' property on or-mwc-input, but basically for any component/function.

There are cases where we put in user specified data like a name.
And in general, displaying a different value than the one given as input can be weird.
Maybe a simple `translate` or `noTranslate` attribute/parameter would help to prevent this?

I saw that some types in `or-mwc-input` are not translated yet.
Would it be a lot of effort to include it in this PR? I guess you tried it out haha

**Comment by MartinaeyNL** [comment]
Clarifying a small discussion we had on the `.` prefix before an attribute (such as using `label=""` instead of `.label=""`);

Putting in a string like `.label="test"` would not work, since it expects a variable. In that case `.label=${'test'}` would work.
So using a regular string attribute is fine, and the best solution here.

https://lit.dev/docs/v2/templates/expressions/#child-expressions

**Comment by DonWillems** [comment]
> In general, I'm struggling with the auto-translating functionality we've added here. On the 'label' property on or-mwc-input, but basically for any component/function.
> 
> There are cases where we put in user specified data like a name. And in general, displaying a different value than the one given as input can be weird. Maybe a simple `translate` or `noTranslate` attribute/parameter would help to prevent this?
> 
> I saw that some types in `or-mwc-input` are not translated yet. Would it be a lot of effort to include it in this PR? I guess you tried it out haha

Your remarks are a bit contradictive. First you state that you don't like everything to be translated, but then you want it applied to all or-mwc-input? Or am I misunderstanding?

Maybe the translate attribute/parameter makes sense indeed. In the case of button labels, lists, and the snackbar they generally won't be user specified data. However, if we translate labels of other inputs, such as text input, the labels might be custom attribute names. So that is where my doubt was in adding the 'auto' translation to the other or-mwc-inputs.

**Comment by MartinaeyNL** [comment]
I should indeed have been more clear about what I mean.

The auto-translating here is good, so I'm not judging that haha
I wanted to clarify that adding an option to disable the translation is necessary and benefitial for cases where there is user input.
So we basically I agree I think

You can PM me to have a call if you want to.

File: ui/component/or-asset-viewer/src/index.ts [hunk 1]
```diff
@@ -499,14 +499,14 @@ function getPanelContent(id: string, assetInfo: AssetInfo, hostElement: LitEleme
                         manager.rest.api.AgentResource.doProtocolAssetImport(asset.id!, fileInfo, undefined, {timeout: 30000})
                             .then(response => {
                                 if (response.status !== 200) {
-                                    showSnackbar(undefined, "Something went wrong, please try again", i18next.t("dismiss"));
+                                    showSnackbar(undefined, "somethingWentWrong", "dismiss");
                                 } else {
-                                    showSnackbar(undefined, "Import successful! Added "+response.data.length+" assets!", i18next.t("dismiss"));
+                                    showSnackbar(undefined, "Import successful! Added "+response.data.length+" assets!", "dismiss");
```
[the remark this answers is not in the document]
Comment (DonWillems) [reply ui/component/or-asset-viewer/src/index.ts:504]: We already have https://github.com/openremote/openremote/issues/1212 for this issue. Could you add some more technical details if needed?

**Review by MartinaeyNL — APPROVED** [review]
Aprroved. Let's keep track of #1212 for future improvements

**Pull request merged**